clap_derive/src/derives/args.rs

```diff
@@ -621,6 +635,24 @@ fn gen_parsers(
             }
         }
 
+        Ty::Array => {
```

Comment:
In #1682
> This change would be breaking if implemented after 3.0 is out, so triaging for 3.0

I'm assuming the breaking case is someone having a parser that produces multiple results from a single value.

Options
- Find a way for people to opt-in
- Put this behind the `unstable-v4` feature flag

Reply:
Done